applications/main/infrared/views/infrared_progress_view.c

```diff
@@ -81,9 +81,9 @@ bool infrared_progress_view_input_callback(InputEvent* event, void* context) {
         if(instance->back_callback) {
             instance->back_callback(instance->context);
         }
+        return true;
```

Comment:
this is a work around because button_menu locked the operation, and i can only either:
1. make an un-lock func (this would change the API version so would be better to do on upstream)
2. make this don't pass even to button menu so it won't lock the button operation.

Reply:
i didnt understand what you meant here at first, but i see it now. ButtonMenu uses `freeze_input` to stop directional inputs while you're holding down a button, but in our case after pressing ok the progress view is shown so ButtonMenu does not receive the release event since the progress view consumes all inputs. and so ButtonMenu remains frozen.

rather than passing all inputs and only consuming short back press events, we could add a case before where if the event involves ok button and release type, it is not consumed, and add a comment explaining this.

or even better, we could make ButtonMenu trigger the click action when Ok is released, rather than when a short press is triggered. then its a small internal behavior change, and API is not impacted :D